doc/refvariable.rst

```diff
@@ -219,7 +219,9 @@ Two possible uses for fixed variables are:
    the sensitivity of the objective with respect to the fixed value may be queried with ``getdual(x)``.
 
 2. For solving a sequence of problems with varying parameters.
-   One may call ``setvalue(x, val)``
+   One may call ``JuMP.fix(x, val)``
```

Comment:
One may call ``JuMP.fix(x,val)`` to change the value the which the variable is fixed or to fix a previously unfixed variable.